File: src/butcher/confy/git.zig
//:_____________________________________________________
//  confy  |  Copyright (C) Ivan Mar (sOkam!)  |  MIT  :
//:_____________________________________________________
//! @fileoverview
//!  Confy : Comfortable and Configurable Buildsystem
//!  @todo : Docs here
//_____________________________________________________|
const Git = @This();
// @deps std
const std = @import("std");
// @deps zstd
const zstd = @import("../lib/zstd.zig");
const cstr = zstd.cstr;

pub const Info = struct {
  baseURL :cstr= "https://github.com/",
  owner   :cstr,
  repo    :cstr,

  pub fn URL(G :*Git.Info) cstr {
    const a = std.fmt.format();
    _ = a;
    std.debug.print("", .{});
    return  G.baseURL;
  }
};

File: src/caller/confy/tools/git.ts
//:______________________________________________________________________
//  ᛝ confy  |  Copyright (C) Ivan Mar (sOkam!)  |  GNU GPLv3 or later  :
//:______________________________________________________________________
// @deps git
import { Octokit } from "@octokit/core"
import { restEndpointMethods } from "@octokit/plugin-rest-endpoint-methods";

const GitHub = Octokit.plugin(restEndpointMethods);
export const gh = new GitHub({})

